packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -161,6 +160,27 @@ abstract contract CallBatch is Script {
         kvs[1][1] = vm.toString(data);
         writeSimpleObject("", "", kvs, true);
     }
+
+    function prank(address router) public {
+        // prank the router itself to avoid governor chain issues
+        vm.prank(router);
+        GovernanceRouter(router).executeGovernanceActions(
+            calls,
+            new uint32[](0),
+            new GovernanceMessage.Call[][](0)
+        );
+    }
+
+    function prankRecovery(address router) public {
+        // prank recovery (only works if recovery is active)
+        address recovery = GovernanceRouter(router).recoveryManager();
+        vm.prank(recovery);
+        GovernanceRouter(router).executeGovernanceActions(
+            calls,
+            new uint32[](0),
```

Comment:
something initially confusing for me about this library is that it's *only* a CallBatch for recovery mode. might be helpful to include that in the documentation (inline, or README) and/or naming. initially confusing because CallBatch in the SDK is a different purpose

Reply:
well, it's ONLY a callbatch for 1 chain. it can be used for non-recovery mode

Reply:
yeah, but the only chain that can be used as 1 chain (Excepting recovery mode) is the Governor router. and when the Governor router is not in recovery mode, it's meant to be able to execute remote calls. so effectively it's a recovery mode CallBatch

Reply:
the current conception is that JSON outputs could be combined out of band

Reply:
i.e. when running for governance, you 


```
$ forge run Script --sig run(string) moonbeam.json
$ forge run Script --sig run(string) ethereum.json
$ jq -s '.[0] * .[1]' ./actions/ethereum.json ./actions/moonbeam.json
```